broker-daemon/orderbook/index.js

```diff
@@ -38,6 +38,9 @@ class Orderbook {
     this.bidIndex = new BidIndex(this.store)
     this.logger = logger
     this.synced = false
+    this._retries = 0
```

Comment:
Can you move the retry logic into its own function or object so as not to muddy the state of the orderbook more generally? It seems like the exponential backoff is independent enough to not need to be mixed in with the other state.